requirements.dev.txt

```diff
@@ -2,35 +2,79 @@
 # This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
-#    pip-compile requirements.dev.in
+#    pip-compile --output-file=requirements.dev.txt requirements.dev.in requirements.in
 #
 appdirs==1.4.4
     # via black
 attrs==21.4.0
     # via pytest
 black==21.7b0
     # via -r requirements.dev.in
+blinker==1.4
+    # via sentry-sdk
```

Comment:
I will remove the annotations on a following PR. I wanted to reduce the number of changes on the PR.